README.md

```diff
@@ -1,148 +1,138 @@
 # POS ICP Integration
 
-## Development environment setup
+## Introduction
 
-```bash
-# yarn installation
-npm install -g yarn
-
-# dependencies installation
-yarn install
-```
-
-## Run formatting and linting
-
-```bash
-npx prettier --write --plugin=prettier-plugin-motoko **/*.mo
-```
-
-## Running the project locally
-
-If you want to test your project locally, you can use the following commands:
-
-```bash
-# Starts the replica, running in the background
-dfx start --background
+### Purpose and Scope
+This project focuses on integrating point of sale (POS) systems with the Internet Computer (ICP) protocol, allowing users to make payments using multiple cryptocurrencies. The goal is to simplify and secure transactions in a decentralized environment, with advanced features such as smart payments and enhanced transaction verification.
 
-# Deploys your canisters to the replica and generates your candid interface
-dfx deploy
-```
+The development of this project has been funded by a grant from the DFINITY Foundation, enabling us to innovate in the area of decentralized payments.
 
-Once the job completes, your application will be available at `http://localhost:4943?canisterId={asset_canister_id}`.
+### Acknowledgments
+We would like to express our gratitude to the following contributors for their invaluable support:
 
-If you have made changes to your backend canister, you can generate a new candid interface with
+- **icp_hub mexico**: For their technical support and for facilitating community engagement, which has been crucial to the progress of the project.
+- **Cafinca**: For implementing and operating our vending machines in Mexico and Chile, providing a real-world environment to test and refine our integration.
```

Comment:
Please put "Cafinca" and "Buidling" by separate. Cafinca provided the "real-world enviroment with coffee machines" and "Buidling" implemented the canister/code.

Well written :)